tests/pom.xml

```diff
@@ -1569,11 +1583,10 @@
                       <SPRING_PROFILES_ACTIVE>${logging.profile}</SPRING_PROFILES_ACTIVE>
                       <JDK_JAVA_OPTIONS>${hono.amqp-adapter.java-options}</JDK_JAVA_OPTIONS>
                       <PN_TRACE_FRM>0</PN_TRACE_FRM>
-                      <JAEGER_SERVICE_NAME>hono-adapter-amqp</JAEGER_SERVICE_NAME>
+                      <JAVA_DEBUG>TRUE</JAVA_DEBUG>
```

Comment:
Should this stay in here?

Reply:
probably not, it doesn't hurt but I will remove it ...